### DISCUSSION TITLE
# [绳网团队]绳网禁止脚本抢委托公告

### DISCUSSION THREAD

**Discussion by lcp1234** [share121/inter-knot#407] [其他]
各位绳网用户：
鉴于近期绳网平台上出现了一些利用脚本自动抢委托的行为，这种行为不仅破坏了公平竞争的环境，还影响了其他用户的正常使用体验。绳网团队对此表示严重关切，并已采取相应措施，以维护平台秩序和用户权益。
绳网是一个以用户为中心的服务平台，我们珍视每一位用户的参与和贡献。为了保护广大用户的利益，绳网团队特别提醒各位用户：
1.  禁止使用任何形式的脚本、插件或自动化工具进行委托的抢取。
2.  一旦发现违规行为，绳网将对相关账户进行警告、限制或封禁处理。
3.  我们鼓励用户积极举报任何违规行为，共同维护一个健康、公平的委托环境。
绳网团队将继续加强监管力度，确保平台的公平性和安全性。我们希望每位用户都能遵守规则，公平竞争，共同创造一个和谐的使用环境。
感谢大家的理解与支持，愿您在绳网的每一次委托都能收获满意的结果！
绳网团队敬上

窥视空洞之人，祝你如愿以偿。

**Comment by InkVoid-0** [comment]
这个委托我接了！

**Reply by Bambook-sth** [reply]
怎么还有贴脸开大的

**Reply by npldqt** [reply]
绳网上单来了

**Comment by lbghhj** [comment]
这个委托接不了

**Comment by npldqt** [comment]
官方呢？治治这些抢委托的脚本啊

**Comment by ShangHuaWenYuan** [comment]
[一❤️个❤️人❤️在❤️家❤️看❤️伊❤️埃❤️斯❤️热❤️舞](https://www.bilibili.com/video/BV1HT421k7Rp/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by kishilong** [comment]
这个委托我接了！（就是脚本

**Comment by Xmug** [comment]
**早该**~~屠屠~~**管管了**

**Comment by WRYYY-S** [comment]
这个委托我接了
欸我怎么接不了

**Comment by kano0x0** [comment]
这个委托我接了！！！
